### ISSUE TITLE
[2.x]: Issue with summary and field name 

### ISSUE THREAD

**Issue by Megafry** [verbb/formie#1937]
### Describe the bug

The field summary doesn't show the subfields `Middle Name` and `Prefix` for the field `Name`

### Steps to reproduce

1. Add the field type name
2. Enable `Use Multiple Name Fields`
3. Enable all fields: `Prefix`, `First Name`, `Middle Name`, and `Last Name` 
4. Add the field summary
5. Fill out the form
6. In the summary the subfield `Middle Name` and `Prefix` are not shown.

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax and Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: Yes or No

### Craft CMS version

4.10.1

### Plugin version

2.1.19

### Multi-site?

No

**Comment by engram-design** [comment]
This is more of a fundamental issue with Name fields, in that the value that's being used for many things like Email Notifications, Integrations, Exports and the Summary field all use the first and last name as their value. There is a "full name" value, but switching things to this would be a breaking change at this point, which will be saved for https://github.com/verbb/formie/issues/1936

Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.19"`.

**Comment by engram-design** [comment]
Fixed in [2.1.20](https://github.com/verbb/formie/releases/tag/2.1.20)

**Comment by sanderpotjer** [comment]
@engram-design this change broke our summary implementation with "Use Multiple Name Fields" set to disabled. 

```
Error: Call to a member function getFullName() on string in /vendor/verbb/formie/src/fields/formfields/Name.php:683
```

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.20"`.

**Comment by engram-design** [comment]
Fixed in [2.1.21](https://github.com/verbb/formie/releases/tag/2.1.21)
